docs/crawler.js

```diff
@@ -0,0 +1,112 @@
+// This is the code example for the crawler configuration file for https://crawler.algolia.com/
```

Comment:
How is this file used? It doesn't seem to be included anywhere.

Reply:
So the code is used at `https://crawler.algolia.com`
Because it defines on how a crawler indexes the docs and how the search after that is working - I descided to leave a reference in our codebase to not to loose it in the future

Reply:
Good thinking, can you add a comment to the file making that explicit. (This file needs to be copied to crawler.algolia.com to work - its stored here as reference...)